src/client.js

```diff
@@ -129,97 +125,69 @@ Client.prototype.connect = function(args) {
 	var config = Helper.getConfig();
 	var client = this;
 
+	var nick = args.nick || "lounge-user";
+
+	var network = new Network({
+		name: args.name || "",
+		host: args.host || "",
+		port: parseInt(args.port, 10) || (args.tls ? 6697 : 6667),
+		tls: !!args.tls,
+		password: args.password,
+		username: args.username || nick.replace(/[^a-zA-Z0-9]/g, ""),
```

Comment:
I know it was already there before but I never paid attention to that `.replace()` statement. It seems weird without further explanation, do you know where that comes from, IRC-something spec or `lounge`-specific decision?
Can we add a comment on that for API documentation purposes?